### PR TITLE
Remove the code in the algorithm that boosts the tweets of Elon

### PR DESCRIPTION
**Pull request by elvodqa**
The title is self-explanatory.

### CODE REVIEW

**Comment by JoshuaLuckers** [comment]
LGTM ;)

**Comment by JuanPablo2655** [comment]
lol

**Comment by josemaia** [comment]
Duplicate https://github.com/twitter/the-algorithm/pull/138

**Comment by sunglocto** [comment]
lgtm

**Comment by elvodqa** [comment]
> Duplicate #138

Damn that was fast.

**Comment by acgfbr** [comment]
merge

**Comment by mutexdeveloper** [comment]
lgtm, no more elon!

**Comment by jplie** [comment]
no more elon ((

**Comment by OffenLord** [comment]
LGTM

**Comment by elvodqa** [comment]
Politics should also be removed in my humble opinion

**Comment by mohammedsahl** [comment]
LGTM! Feel free to merge

**Comment by SaquibShahzad** [comment]
LGTM

**Comment by elvodqa** [comment]
Done!!! Any devs willing to check my pr?

**Comment by bchurton** [comment]
lgtm

**Comment by NateShoffner** [comment]
LGTM

**Review by bee-san — COMMENTED** [review]
Interesting. Looking into it.

**Review by MickaelCa — COMMENTED** [review]
LGTM

**Comment by garrettsutula** [comment]
LGTM ✅

**Review by acarlson99 — COMMENTED** [review]
LGTM, perhaps consider removing other references to ElonFeature :)

https://github.com/twitter/the-algorithm/search?q=DDGStatsElonFeature

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala [hunk 1]
```diff
@@ -221,30 +221,14 @@ object HomeTweetTypePredicates {
...
-    (
-      "author_is_republican",
-      candidate =>
-        candidate
-          .getOrElse(AuthorIdFeature, None)
-          .exists(candidate.getOrElse(DDGStatsRepublicansFeature, Set.empty[Long]).contains)),
+          .exists(candidate.getOrElse(DDGStatsVitsFeature, Set.empty[Long]).contains))
   )
-
+  // no more politics 😭😭😭
```
Comment (s-ven) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:232]: thank you

**Comment by bchurton** [comment]
Personally I think we should just clear the entire codebase and just `rm -rf` the site. Simple, effective, and removes all problems.

**Comment by martynix** [comment]
:))))

**Comment by dromzeh** [comment]
LGTM

**Comment by jared-verta** [comment]
Approved. Ship it.

**Comment by katlol** [comment]
posting in an epic thread

**Comment by ryanbnl** [comment]
Someone give me merge rights and we're go!

**Comment by cjoelrun** [comment]
Can someone explain how is this used in the algorithm? Looks like it sets up a predicate for Elon that's only later referenced in metrics:
https://github.com/twitter/the-algorithm/blob/7f90d0ca342b928b479b512ec51ac2c3821f5922/home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/feature_hydrator/RequestQueryFeatureHydrator.scala#L86-L95

**Comment by thiagopos** [comment]
Merge this!

**Comment by mar-pan** [comment]
awaited feature, please ship it 🥳

**Comment by Scribilicious** [comment]
When will it be merged?

**Comment by rguzg** [comment]
LGTM

**Comment by aobr95** [comment]
haha or delete Elon account

**Comment by harrrshall** [comment]
elon be like gjb bezzati hai

**Comment by amrityam** [comment]
LGTM

**Comment by jangels** [comment]
🤩, LGTM, lol, mask angry

**Pull request closed without merging**